prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -293,8 +298,6 @@ Examples:
         val coolTestList = listOf(
             Ping(),
             SftpcheckTest(),
-            End2End(),
```

Comment:
Seemed to make more sense to put these at the end to get earlier signal if one of the other tests fail